tests/test_convert_lit_checkpoint.py

```diff
@@ -101,3 +101,47 @@ def test_against_original_gpt_neox():
     ours_y = ours_model(x)
     theirs_y = theirs_model(x)["logits"]
     torch.testing.assert_close(ours_y, theirs_y)
+
+
+def test_maybe_unwrap_statedict(tmp_path):
+    import lightning as L
+    import torch
+    from lit_gpt import Config, GPT
+    from scripts.convert_lit_checkpoint import convert_lit_checkpoint
+    from finetune.full import save_checkpoint
+
+    # fabric is needed for finetune.full::save_checkpoint
+    fabric = L.Fabric(devices=1)
+
+    ckpt_path: Path = tmp_path / "lit_model_finetune.pth"
+    ckpt_name = ckpt_path.name
+
+    model_name = "pythia-1b"
+    ours_config = Config.from_name(
+        model_name,
+        block_size=2048,
+        n_layer=2,
+        n_embd=2048,
+        n_head=8,
+        padding_multiple=128,
+    )
+    ours_model = GPT(ours_config)
+
+    # save checkpoint and check for model key
+    save_checkpoint(fabric, ours_model, ckpt_path)
+    statedict_with_model_key = torch.load(ckpt_path)
+    assert statedict_with_model_key.get("model")
+    assert len(statedict_with_model_key) == 1
+
+    # convert and check that model key does not exist
+    # and that a known key for pythia does exists
```

Comment:
```suggestion
    # and that a known key for pythia exists
```

Reply:
implemented in [169acf6](https://github.com/Lightning-AI/lit-gpt/pull/341/commits/169acf6aed0f50e4f6c929274a7902c46bb4815f)